lib/stat_tracker.rb

```diff
@@ -1,30 +1,21 @@
 require "csv"
+require "./lib/team"
+require "./lib/game"
+require "./lib/game_team"
 
 class StatTracker
   attr_reader :teams, :games, :game_teams
 
-  def initialize(team_path, game_path, game_teams_path)
-    @teams = read_teams_from_csv(team_path)
-    @games = read_games_from_csv(game_path)
-    @game_teams = read_game_teams_from_csv(game_teams_path)
+  def initialize(teams, games, game_teams)
+    @teams = teams
+    @games = games
+    @game_teams = game_teams
   end
 
-  def self.from_csv(locations)
-    self.new(locations[:teams], locations[:games], locations[:game_teams])
+  def self.from_csv(locations = {games: './data/games_sample.csv', teams: './data/teams_sample.csv', game_teams: './data/game_teams_sample.csv'})
+    Game.from_csv(locations[:games])
+    Team.from_csv(locations[:teams])
+    GameTeam.from_csv(locations[:game_teams])
+    self.new(Team.all_teams, Game.all_games, GameTeam.all_game_teams)
```

Comment:
This looks good - addresses my initial question posed in our slack channel about initiating the games, teams, and game_teams classes with the StatTracker.from_csv method.